src/views/Incorporation/IncorporationAgreement.vue

```diff
@@ -70,7 +70,7 @@
             </ul>
           </div>
 
-          <template v-if="!isTypeBcCcc">
+          <template v-if="isTypeBcomp">
```

Comment:
Should this not be for CCC and ULC's?

Reply:
Based on updated text, this is only for Ben Companies.

Reply:
You are correct, I miss interpreted the meaning of this text this.